web/e2e-tests/mention.test.ts

```diff
@@ -39,4 +39,21 @@ async function test_mention(page: Page): Promise<void> {
     await common.check_messages_sent(page, "zhome", [["Verona > Test mention all", ["@all"]]]);
 }
 
+async function test_private_mention(page: Page): Promise<void> {
```

Comment:
nit: Maybe a more specific test name as to the eventual action it does. E.g., in this case, you are testing if the private_mention changes to silent_mention.

Reply:
Changed to `test_change_private_mention_to_silent_mention`